web-animations-2/Overview.bs

```diff
@@ -2006,6 +2045,34 @@ partial interface AnimationTimeline {
 
 </div>
 
+<h3 id="the-animation-interface">The <code>Animation</code> interface</h3>
+<pre class="idl">
+[Exposed=Window]
+partial interface Animation : EventTarget {
+    attribute CSSNumberish?       startTime;
+    attribute CSSNumberish?       currentTime;
+};
```

Comment:
I think this should be expressed as a delta to the Animation interface from web-animations-1. E.g.

Update the Animation interface to the following:

> Repeat of complete interface with members changed

Alternately, we could try to target the specific members by saying update the startTime and currentTime of the Animation interface to the following: